### ISSUE TITLE
$refs.colgrouptable undefined in vsTable 

### ISSUE THREAD

**Issue by percymamedy** [lusaxweb/vuesax#634]
### System:

- OS:  Ubuntu 19.04
- Node.js version: v11.14.0
- Vuesax version: 3.9.0
- Browser version: 0.66.101 Chromium: 75.0.3770.142 (Official Build) (64-bit)
-  Which package manager did you use for the install? : Yarn

### Details:

In vsTable line 78 colgrouptable has been commented out in favor of ```<slot></slot>```

https://github.com/lusaxweb/vuesax/blob/master/src/components/vsTable/vsTable.vue#L78

However in the method ```changeTdsWidth``` of the Component ```this.$refs.colgrouptable``` is referenced :

https://github.com/lusaxweb/vuesax/blob/master/src/components/vsTable/vsTable.vue#L403

This is causing an error whenever window is resized on the page containing vsTable component.

```
Uncaught TypeError: Cannot read property 'querySelectorAll' of undefined
    at VueComponent.changeTdsWidth (vuesax.common.js?fb9a:10055)
    at VueComponent.listenerChangeWidth (vuesax.common.js?fb9a:10037)
```

### Steps to reproduce:

1)  Add a vsTable component on a page and set a v-model attribute on the component.
2) Open browser console.
3) Resize browser window.

**Comment by jd-0001** [comment]
Please update vuesax. Also check [PR](https://github.com/lusaxweb/vuesax/pull/655).

**Comment by percymamedy** [comment]
Hello, @jd-0001 

Thank you for you reply. Could you please specify which version of VueSax contains this fix ?

As mentioned above the version am using is 3.9.0

**Comment by jd-0001** [comment]
this is fixed from version 3.10.0 please run **yarn upgrade vuesax** or **npm update vuesax**
